### PR TITLE
Report installed packages on COS

### PR DESCRIPTION
**Pull request by edjee**
Passed gofmt, goimports, goline, go vet, go build, go test.
The resulted binary worked as expected on a COS instance.

### CODE REVIEW

File: packages/packages.go [hunk 1]
```diff
@@ -64,6 +66,7 @@ type Packages struct {
 	Deb           []PkgInfo     `json:"deb,omitempty"`
 	Zypper        []PkgInfo     `json:"zypper,omitempty"`
 	ZypperPatches []ZypperPatch `json:"zypperPatches,omitempty"`
+	Cos           []PkgInfo     `json:"cos,omitempty"`
```
Comment (adjackura) [inline packages/packages.go:69]: We aren't super diligent about this, but I assume you may prefer COS over Cos

File: packages/packages_linux.go [hunk 2]
```diff
@@ -127,6 +127,9 @@ func GetInstalledPackages(ctx context.Context) (Packages, error) {
 			pkgs.Deb = deb
 		}
 	}
+	if cos, err := InstalledCOSPackages(); err == nil {
```
Comment (adjackura) [inline packages/packages_linux.go:130]: Your missing the check for COSPkgInfoExists

**Review by adjackura — CHANGES_REQUESTED** [review]

File: packages/packages.go (hunk 1 above)
Comment (edjee) [reply packages/packages.go]: Fixed in the new commit.

File: packages/packages_linux.go (hunk 2 above)
Comment (edjee) [reply packages/packages_linux.go]: Fixed in the new commit.

By the way, the previous commit I made seems lost. How can I preserve the patch history, like keeping the past "patch sets" for a specific CL in Gerrit?

File: packages/cos.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import (
+	"fmt"
+	"strconv"
+
+	"cos.googlesource.com/cos/tools.git/src/pkg/cos"
+	"github.com/GoogleCloudPlatform/osconfig/osinfo"
+)
+
+// COSPackageInfoExists returns whether COS package information exists.
+func COSPackageInfoExists() bool {
```
Comment (adjackura) [inline packages/cos.go:26]: Leave this as a var in packages.go and populated it in an init() function in this file. That way this check is only done once not every run, and it's a bit simpler to test.

Comment (edjee) [reply packages/cos.go]: Done in the second commit.

**Review by adjackura — APPROVED** [review]

**Pull request merged**